k8s/operator/values.yaml

```diff
@@ -95,6 +107,12 @@ sdp:
       pullSecrets: []
       pullPolicy: Always
 
+    serviceAccount:
+      create: true
+      name:
```

Comment:
Do we need an empty string here?

Reply:
I dont think it's needed since we just do if there but I think it's nice to have it explicit there.